src/transaction/log_recovery.c

```diff
@@ -2497,7 +2188,6 @@ log_recovery_analysis (THREAD_ENTRY * thread_p, LOG_LSA * start_lsa, LOG_LSA * s
 		       LOG_LSA * end_redo_lsa, bool is_media_crash, time_t * stop_at, bool * did_incom_recovery,
```

Comment:
Maybe the comments/notes to these functions whose internal mechanics has been changed should be revised too?

Eg:
```
since the last checkpoint record reflected in the log and the data volumes
```

Reply:
The comment does not look outdated to me. The quoted mentions is still true because the last checkpoint is still reflected into the log & data volumes. An implementation detail has changed, but in my opinion it is not useful to mention it in the description comment of log_recovery_analysis.